### ISSUE TITLE
Easy way to select all metrics under metrics explorer page

### ISSUE THREAD

**Issue by Sharathmk99** [aimhubio/aim#1999]
## 🚀 Feature

When we have many metrics like more than 100, it's really difficult to select metrics one by one. Can we have option "Select All" which helps to select and deselect all metrics at once.

![image](https://user-images.githubusercontent.com/3970340/180882402-7037470f-8546-4544-a2f5-2b7775617b24.png)

When we have long metrics name, it's not readable of which metrics is been selected.

**Comment by roubkar** [comment]
Hey @Sharathmk99, thanks for opening this issue, currently we're exploring/designing a new UX for the Query Form which, will keep you updated with the progress, also will try to address the issue with long metric names.

**Comment by Sharathmk99** [comment]
@roubkar Thank you. Is there any doc where I can learn about new UX design?

**Comment by gorarakelyan** [comment]
@Sharathmk99 here are the prototypes of the new explorer design:

### Prototype 1

https://www.figma.com/proto/rVjgCyGwvgaDZ8kXPyPw2m/PoC_version-1?page-id=0%3A1&node-id=1%3A2018&viewport=489%2C426%2C0.49&scaling=min-zoom&starting-point-node-id=1%3A2018

<img width="1328" alt="image" src="https://user-images.githubusercontent.com/13848158/180972254-01dc9e99-5851-4c6a-9159-c5e4a9db2c2a.png">

### Prototype 2

https://www.figma.com/proto/zGFAW892lLfeH6WDleKpGH/PoC_version-2?page-id=0%3A1&node-id=1%3A3910&viewport=486%2C500%2C0.82&scaling=min-zoom&starting-point-node-id=1%3A3910

<img width="1324" alt="image" src="https://user-images.githubusercontent.com/13848158/180972350-5bae5399-2af1-492a-b01c-c47a21031829.png">

Would appreciate your feedback on this:
- Do you have questions or things that are not clear?
- Which one is more intuitive?
- How is it compared to the current version of Aim?
- Which one would delight you the most as a user?

**Comment by gorarakelyan** [comment]
@Sharathmk99 btw edit mode might be very helpful here. It enables writing arbitrary python conditions to filter by any property of runs and/or metrics. For example:
- `True` (select all the metrics of all runs)
- `run.hparams.learning_rate < 0.001` (select all the metrics of runs which learning_rate is less than 0.001)
- `metric.name == 'loss'` (select all loss metrics)

To turn on the edit mode, click on the pencil icon located on the top right corner:
![Screen Shot 2022-07-26 at 14 05 34](https://user-images.githubusercontent.com/13848158/180980886-de113e6e-6c10-4b98-ba40-c0daaa653bbc.png)
